AmazonChimeSDK/AmazonChimeSDK/internal/audio/protocols/AudioClientProtocol.swift

```diff
@@ -10,7 +10,17 @@ import AmazonChimeSDKMedia
 import Foundation
 
 @objc public protocol AudioClientProtocol {
-    func startSession(_ host: String!, basePort port: Int, callId: String!, profileId: String!, microphoneMute mic_mute: Bool, speakerMute spk_mute: Bool, isPresenter presenter: Bool, sessionToken tokenString: String!, audioWsUrl: String!, callKitEnabled: Bool) -> audio_client_status_t
+    // swiftlint:disable function_parameter_count variable_name
+    func startSession(_ host: String!,
+                      basePort port: Int,
+                      callId: String!,
+                      profileId: String!,
+                      microphoneMute mic_mute: Bool,
+                      speakerMute spk_mute: Bool,
+                      isPresenter presenter: Bool,
+                      sessionToken tokenString: String!,
+                      audioWsUrl: String!,
+                      callKitEnabled: Bool) -> audio_client_status_t
 
```

Comment:
You can probably reenable `function_parameter_count` and `variable_name` here.

Reply:
> You can probably reenable `function_parameter_count` and `variable_name` here.

Added `swiftlint:enable` here

Reply:
Does this mean for every protocol files, we have to disable swiftlint? Wouldn't it better to ignore it by using swiftlint.yml?

Reply:
> Does this mean for every protocol files, we have to disable swiftlint? Wouldn't it better to ignore it by using swiftlint.yml?

Only for functions that violates the rule, in this case the parameter length way exceeds its recommended 5. And variable name `mic_mute` contains not allowed characters. We cannot change the function signature in this case because it's determined by the media client interface so had to add this exception. I think it still makes sense to have those rules enabled for code we own. What do you think?